src/_common/comment/widget/widget.ts

```diff
@@ -264,6 +269,14 @@ export default class AppCommentWidget extends Vue {
 			throw new Error(`You need a store set before fetching comments.`);
 		}
 
+		// Check to see if we received 'initialTab' as a prop. If so, we want to setSort()
+		// to filter comments based on that parameter. This allows us to set the comment
+		// sorting to the "You" tab when you leave a comment from an event item.
+		if (this.initialTab && !this.tabSet) {
```

Comment:
If you move this into init() then you don't need tabSet, since it'll only trigger on created() or when model changes.